synapse/module_api/__init__.py

```diff
@@ -1452,6 +1452,65 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def create_room(
+        self,
+        user_id: str,
+        config: JsonDict,
+        ratelimit: bool = True,
+        creator_join_profile: Optional[JsonDict] = None,
+    ) -> Tuple[dict, int]:
+        """Creates a new room.
```

Comment:
Please add a note about the version of Synapse this was added in (likely 1.66.0 since Synapse 1.65.0rc1 is coming out today).

Reply:
Sorry, I forgot to do this, thanks. Done here: 8116429